data/geany.glade

```diff
@@ -3498,6 +3516,7 @@
                                         <property name="visible">True</property>
                                         <property name="can_focus">True</property>
                                         <property name="tooltip_text" translatable="yes">Display height in rows for the autocompletion list</property>
+                                        <property name="invisible_char">●</property>
```

Comment:
I'm using Glade 3.8.5, or glade-gtk2, on Arch Linux. Anyways, the invisible character used seems to depend on the current font you're using. Source code comments confirm this.
https://github.com/GNOME/gtk/blob/gtk-2-24/gtk/gtkentry.c#L6907